specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+
+ChargeSubcategory MUST abide by the following requirements:
+
+- The ChargeSubcategory MUST be present in the billing data.
+- ChargeSubcategory is of type String and MUST be one of the allowed values.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Usage" and the charge is covered by a commitment.
+  - When a usage charge is covered by a commitment, ChargeSubcategory MUST be "Commitment Used".
+  - When a commitment is not used within the committed period, ChargeSubcategory MUST be "Commitment Not Used" for the unused usage charge.
+- ChargeSubcategory MUST be null when ChargeType is "Usage" and is not covered by a commitment.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Adjustment".
+  - When ChargeSubcategory is "Refund" or "Credit", the charge MUST be negative.
+  - When an adjustment applies to a specific item, the corresponding FOCUS columns that identify that item MUST NOT be null and MUST match the applicable item details the adjustment pertains to.
+- ChargeSubcategory MUST be null when ChargeType is "Purchase" or "Tax".
+
+## Column ID
+
+ChargeSubcategory
+
+## Display Name
+
+Charge Subcategory
+
+## Description
+
+Indicates what kind of usage or adjustment the row represents.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | True           |
+| Value format    | list-of-values |
+
+Allowed values for usage:
+
+| Value             | Description                                                                                                                                                                                                                                                                                                                                                |
+| :---------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Commitment Used   | Usage charges that are covered by pre-existing commitments, such as capacity commitments or commitment-based discounts.                                                                                                                                                              |
+| Commitment Not Used | Amortized usage charges for the portion of a commitment that has not been used. For example, if an organization has a commitment-based discount that is not fully utilized, the unused portion falls under this category. It highlights an area where the organization is not fully leveraging its commitments, which could be a lost cost-saving opportunity. |
```

Comment:
I understand there are 2 cases for charges with Commitment Not Used:
- The charge is for the unused portion of a commitment. The charge must be present for every commitment. The value MUST be 0 if the commitment is fully used. I would not elaborate on the potentially lost cost-saving opportunity, because a commitment is beneficial as long as the effective cost is lower than the negotiated cost (on-demand equivalent cost) over the commitment period.
- The charge is for a SKU that is billed at the on-demand rate. This is currently not documented?

Is it correct?

Reply:
Maybe it would be clearer to have 3 values for Usage:
- No commitment: usage billed at on-demand rate
- Commitment Used: usage covered by a commitment
- Commitment Not Used: unused commitment

Reply:
I think we have a blank or null value for "no commitment" that represents the usage is billed at on-demand rate.

Reply:
IMHO, it would be clearer and better for filtering and for reporting to introduce an explicit value for Charge Sub Category for Usage when it relates to no commitment.

Reply:
> The charge is for the unused portion of a commitment. The charge must be present for every commitment. The value MUST be 0 if the commitment is fully used.

The Discount Handling attribute might be a better place to convey the intent of the "Not Used" row. That row is only there _if_ there's an unused amount. Commitments that are fully utilized will not have an extra row.

> The charge is for a SKU that is billed at the on-demand rate. This is currently not documented?

Not exactly. This is where showing data would be helpful. The "Not Used" row is tied to the commitment, but not any usage (because there is no usage).

> IMHO, it would be clearer and better for filtering and for reporting to introduce an explicit value for Charge Sub Category for Usage when it relates to no commitment.

Agreed. We were keeping it null because we think there could be multiple values. If you only care about on-demand, then you can get that out of PricingModel.

Reply:
> Commitments that are fully utilized will not have an extra row.

As a practitioner, I would find very convenient that are all commitments have an unused commitment charge, so that I can easily calculate every commitment utilization from FOCUS data and I find it's also better for transparency ("zero is a number like others"). The absence of unused commitment charge may also denote a data error.

Reply:
Null works here because you can then use the null value with commitment charges value to determine unused value.

Reply:
if all agree @marc-perreaut @flanakin  i will resolve this as we keep null

Reply:
@thecloudman  - I'm resolving this conversation in the VS2 meeting